types/datatables.net-rowgroup/index.d.ts

```diff
@@ -83,6 +83,6 @@ declare namespace DataTables {
         /**
          * Provide a function that can be used to control the data shown in the start grouping row
          */
-        startRender?: (rows: Api, group: string) => string|HTMLElement|JQuery;
+        startRender?: (rows: Api, group: string, level: number) => string|HTMLElement|JQuery;
```

Comment:
This appears to be new in 1.1.  Do you want to update the version number in the header?

Reply:
Done, thanks for the hint